relayertest/legacy.go

```diff
@@ -0,0 +1,96 @@
+package relayertest
+
+import (
+	"bytes"
+	"fmt"
+	"runtime"
+	"strings"
+	"sync"
+
+	"github.com/strangelove-ventures/ibc-test-framework/ibc"
+)
+
+// GetLegacyTestCase looks up the given testCase and returns a function suitable for the existing cmd structure.
+func GetLegacyTestCase(testCase string) (func(string, ibc.ChainFactory, ibc.RelayerImplementation) error, error) {
+	var testFunc func(t TestingT, cf ibc.ChainFactory, rf ibc.RelayerFactory)
+	switch testCase {
+	case "RelayPacketTest":
+		testFunc = TestRelayer_RelayPacket
+	default:
+		return nil, fmt.Errorf("no test case exists for %s", testCase)
+	}
+
+	return func(testName string, cf ibc.ChainFactory, ri ibc.RelayerImplementation) error {
+		w := &tWrapper{
+			name: testName,
+			cf:   cf,
+			rf:   ibc.NewBuiltinRelayerFactory(ri),
+		}
+		return w.run(testFunc)
+	}, nil
+}
+
+// tWrapper satisfies TestingT for use in a non-test binary.
+type tWrapper struct {
```

Comment:
I under-documented the `tWrapper` type for this draft. This acts like a `*testing.T` but can be used in a plain, non-test Go package.